plugins/VstEffect/VstEffectControls.cpp

```diff
@@ -291,13 +291,14 @@ void VstEffectControls::rolrPreset()
 void VstEffectControls::selPreset()
 {
 
-     QAction *action = qobject_cast<QAction *>(sender());
```

Comment:
Not sure I understand, but I ran clang-tidy with --format-style=file, so I figured it would make the necessary formatting changes. Looks like it removed the indent, but do you mean to keep it or something different?

Reply:
Hmm the github editor made it look worse than it actually is. Anyways, I think `if(action)` must have `{}` due to our coding rules. Though you can also join both `if`-statements together.

Anyways, I can also live with how it is now. I just think it looks a bit weird.

Reply:
> Hmm the github editor made it look worse than it actually is. Anyways, I think `if(action)` must have `{}` due to our coding rules. Though you can also join both `if`-statements together.
> 
> Anyways, I can also live with how it is now. I just think it looks a bit weird.

Ah, I see. At first I thought you were just referring to just the ``QAction *action = qobject_cast<QAction *>(sender());`` line. Looks like the code that follows has always been misformatted that way then. I don't think it would hurt for me to fix it and join the two ``if``-statements though.